maintainers/maintainer-list.nix

```diff
@@ -10604,6 +10604,12 @@
     githubId = 11824817;
     name = "Marti Serra";
   };
+  xworld21 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add xworld21`` and sort it alphabetically.

Reply:
I believe I put myself in the right spot (between `xv*` and `xwv`), it's the people after `zzamboni` who are not in order! I will fix the commit message in the final squash.

Reply:
Yeah some entries slipped through.